core/src/main/scala/org/apache/spark/h2o/utils/H2OContextUtils.scala

```diff
@@ -60,5 +62,18 @@ private[spark] trait H2OContextUtils extends Logging{
   def isRunningOnCorrectSpark(sc: SparkContext) = sc.version.startsWith(BuildInfo.buildSparkMajorVersion)
 
 
+  def withConversionDebugPrints[R <: Frame](sc: SparkContext, conversionName: String, block: => R): R = {
+    val sparkTesting = sc.getConf.getOption("spark.testing").orElse(sys.props.get("spark.testing"))
+
+    if (sparkTesting.nonEmpty && sparkTesting.get.toBoolean) {
```

Comment:
one more note: i would use different spark property then spark.testing - just to avoid different code path - i would say `spark.h2o.measurements.timing`

Reply:
good point!